docx/helper/helper.go

```diff
@@ -25,3 +27,43 @@ func GenerateXML(wordDoc *docx.Docx) (xmlDoc *xml.XmlDocument, err error) {
 	bytes := []byte(content)
 	return ParseXML(bytes)
 }
+
+// FillParagraph inserts the given content into the provided docx XML paragraph node. Note that newlines aren't respected - you'll need to create a new paragraph node for each.
```

Comment:
Totally. Thinking most of this package should move over, but would rather think about that separately.